src/main/kotlin/dev/arbjerg/lavalink/client/LavalinkNodeOptions.kt

```diff
@@ -0,0 +1,39 @@
+package dev.arbjerg.lavalink.client
+
+import dev.arbjerg.lavalink.client.loadbalancing.IRegionFilter
+import dev.arbjerg.lavalink.internal.TIMEOUT_MS
+import java.net.URI
+
+data class LavalinkNodeOptions(val name: String,
```

Comment:
Constructor can be private & I think `NodeOptions` might be a better class name

Reply:
We can private the constructor, but the constructor will still be exposed publicly through the generated copy function, is that OK with you? It's a better state than now in terms of hiding the constructor but in theory if someone is copying node options they can still access the private constructor.

Reply:
Yeah that's ok with me, I want to encourage users to use the builder instead of the constructor